builder/Dockerfile

```diff
@@ -12,16 +12,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.16.5-alpine AS builder
+FROM golang:1.17.6-alpine AS builder
 
 ARG GIT_TAG
-
+WORKDIR /src
```

Comment:
@jcmoraisjr small additional change, it's preferred over using cd

Reply:
sure, sounds good.